frontend/projects/ui/src/app/components/form-object/form-object.component.ts

```diff
@@ -63,6 +64,37 @@ export class FormObjectComponent {
   ) {}
 
   ngOnInit() {
+    this.setDisplays()
+
+    // setTimeout hack to avoid ExpressionChangedAfterItHasBeenCheckedError
+    setTimeout(() => {
+      if (this.original) {
+        Object.keys(this.current || {}).forEach(key => {
+          if ((this.original as Config)[key] === undefined) {
+            this.hasNewOptions.emit()
```

Comment:
Do we need it for every new option?

Reply:
No, if any options are new, we can emit and abort